varats-core/varats/provider/bug/bug_provider.py

```diff
@@ -45,9 +45,15 @@ def find_all_pygit_bugs(self) -> tp.FrozenSet[bug.PygitBug]:
         Returns:
             A set of PygitBugs.
         """
+        resulting_bugs: tp.Set[bug.PygitBug] = set()
         if self.__github_project_name:
-            return bug.find_all_pygit_bugs(self.__github_project_name)
-        return frozenset()
+            resulting_bugs.union(
+                bug.find_all_issue_pygit_bugs(self.__github_project_name)
+            )
+        resulting_bugs.union(
+            bug.find_all_commit_message_pygit_bugs(self.project.SOURCE)
```

Comment:
This is actually does not typecheck. I do not know why mypy does not detect this.

Also, from what I see, it is sufficient to only pass the project name here. You should change that for all functions. Remember to also change the type of the parameters to `str` then.

```suggestion
            bug.find_all_commit_message_pygit_bugs(self.project.NAME)
```